src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -64,8 +64,9 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
                     cpu=DEFAULT_CPU,
                     timeout=DEFAULT_TIMEOUT_IN_SEC,
                     values=None,
-                    source_trigger_name='defaultSourceTriggerName',
-                    commit_trigger_enabled=True,
+                    source_trigger_name=None,
```

Comment:
Revert to use default name here. That comes in handy when user does `-h` on the command.

Reply:
Changed in most recent update.